app/controllers/print_jobs_controller.rb

```diff
@@ -4,8 +4,14 @@
 # Pretty simple wrapper for print job, and incredibly un-opinionated, simply passes
 # the parameters straight through to the print job.
 class PrintJobsController < ApplicationController
-  def create
+  def create # rubocop:todo Metrics/AbcSize
```

Comment:
Do you plan to fix this rubocop:todo or leave until later?

Reply:
👍 ill sort